SRS.md

```diff
@@ -1,28 +1,98 @@
+  * User enters employee’s data and presses “Cancel” button;  
+  * Data don’t save in database, then list of employees' records is displaying to user.  
+  * If the user selects the menu item "Departments” or "Employees", the data will not be saved to the database and the corresponding form with updated data will be opened. 
+   
+   ![image](https://user-images.githubusercontent.com/83345134/127285174-8b015c12-a1b2-4ff9-a32c-d93a516a4bbb.png "Add employee")  
+      Pic 2.2 Add employee.  
+      
+  __When adding a employee, the following details are entered:__  
+  * Department – employee’s work place;  
+  * Name – employee’s name;  
+  * Salary – employee’s salary;  
+  * Date of birth – employee’s date of birth.  
 
-
-One of the site feature is managing employee resourses. This feature will take "employees" tab in main menu. The following tab requires big table for 
-describing every employee and button "add" - for adding new eployees.  
-
-On the following picture situated eployee's tab:  
-
-![image](https://user-images.githubusercontent.com/83345134/127285130-2928a7fa-dd6e-4b84-a077-ff80b723aa25.png "Employees list")
-
-
-By clicking green button user will add new employee. To add new employee user should write down his department, some personal data and his salary.
-Then user should press green button "save" to store this data in database.  
-
-![image](https://user-images.githubusercontent.com/83345134/127285174-8b015c12-a1b2-4ff9-a32c-d93a516a4bbb.png "Add employee")
-
-To edit enployee's information the user should double click employee in the table, fill in the required info and press "Yes", otherwise press "Cancel" to return back: 
-
-![image](https://user-images.githubusercontent.com/83345134/127285373-3f14cb74-9510-4dc3-8e3a-2c2c01dee4a4.png "Edit employee")
-
-To delete an employee user should right-click eployee in the table and press delete button. Otherwise, hi will need to press "Cancel" button to return back to employees' list:
-
-![image](https://user-images.githubusercontent.com/83345134/127285318-fc4847d6-ff49-4f51-863a-19ffafd29391.png "Delete employee")
+  __Constraints for data validation:__  
+  * Department – one of the departments;  
+  * Name –  maximum length of 45 characters;  
+  * Salary – maximum value is 2000;  
+  * Date of birth – date in format dd/mm/yyyy.   
+      
+  ### 2.3 Edit employee  
+  __Main scenario:__  
+  * User clicks the “Edit” button in the employees list view mode;  
+  * Application displays form to enter employee data;  
+  * User enters cemployee’s data and presses “Save” button;  
+  * If any data is entered incorrectly, incorrect data messages are displayed;  
+  * If entered data is valid, then edited data is added to database;  
+  * If error occurs, then error message is displaying;  
+  * If employee’s record is successfully edited, then list of employees with added records is displaying.  
+ 
+  __Cancel operation scenario:__  
```

Comment:
I think this scenario should be described in a more general way, as it applies to all actions: "Add", "Edit" and "Delete".